README.md

```diff
@@ -63,6 +63,5 @@ To run all integration tests, run:
 
 ## Known issues and limitations
 
-* Only one (and exactly one) script may be defined in the `pyproject.toml`
-  file. Zero scripts, or multiple scripts, will result in the buildpack failing
-  detection and therefore not participating in the order group.
+* When `BP_LIVE_RELOAD_ENABLED` is not set, only one (and exactly one) script may be defined in the `pyproject.toml` file.
```

Comment:
I think this is the wrong environment variable - I think it should be `BP_POETRY_RUN_TARGET`.